deploy/static/provider/kind/deploy.yaml

```diff
@@ -339,7 +339,6 @@ spec:
             - --validating-webhook-certificate=/usr/local/certificates/cert
             - --validating-webhook-key=/usr/local/certificates/key
             - --publish-status-address=localhost
-            - --watch-ingress-without-class
```

Comment:
this also not needed anymore?

Reply:
I was very confused with this change as well, but it comes from the generator in `hack/generate-deploy-scripts.sh`. I assume there was a change in the mainline, that wasn't followed by the re-generation of the static providers.

In fact, though, the recent merge from https://github.com/kubernetes/ingress-nginx/pull/7709/files#diff-f3875840bc792db4552edd11518c98846b1ab25e21930492aeb454a1fa02cb79L342 already removed this line.